### PR TITLE
Fix quantity overflow

### LINKED ISSUE #145: quantity overflow
quantity overflow *resulting* from addq not *specially* accounted for

### PR DESCRIPTION
**Pull request by justacasul**
fixed quantity overflow

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddQuantityToItemCommand.java [hunk 1]
```diff
@@ -35,15 +35,15 @@
     public static final String MESSAGE_ITEM_NOT_PROVIDED = "Item name must be provided.";
     public static final String MESSAGE_QUANTITY_NOT_PROVIDED = "Quantity must be provided.";
     public static final String MESSAGE_NEGATIVE_QUANTITY = "Quantity will become negative.";
-
+    private static final String MESSAGE_OVERFLOW_QUANTITY = "Quantity will overflow.";
```
Comment (blackonyyx) [inline src/main/java/seedu/address/logic/commands/AddQuantityToItemCommand.java]: Needs a test case for this

**Review by Rahul0506 — APPROVED** [review]
LGTM

**Review by blackonyyx — COMMENTED** [review]
LGTM after unittest added

File: src/main/java/seedu/address/logic/commands/AddQuantityToItemCommand.java (hunk 1 above)
Comment (justacasul) [reply src/main/java/seedu/address/logic/commands/AddQuantityToItemCommand.java]: yee 
image

**Pull request merged**